tests/playwright-test/golden.spec.ts

```diff
@@ -696,12 +696,16 @@ test('should respect comparator in config', async ({ runInlineTest }) => {
   expect(result.report.suites[0].specs[0].tests[1].status).toBe('unexpected');
 });
 
+// ../../snapshot!.txt as a snapshot name.
+// Its different per platform because we normalize the slashes.
+const kSnapshotNameWithSpecialCharactersOnDisk = process.platform === 'win32' ? 'snapshot-e636e8.txt' : 'napshot-6320a2.txt';
```

Comment:
needed for https://github.com/microsoft/playwright/blob/5821c547aacfd542259c8a9707b1eee119f0b692/packages/playwright-test/src/util.ts#L212-L218 which normalises the slashes for Windows.